addons/languages/rusyn/pack/src/main/res/xml/rusyn_dictionaries.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Dictionaries>
+    <Dictionary
+        autoTextResourceId="@xml/rusyn_autotext"
+        description="@string/rusyn_dictionary_description"
+        dictionaryResourceId="@array/rusyn_words_dict_array"
+        id="a6cc6503-e08d-499d-9f0c-4471a67b3258"
```

Comment:
this id is already used. Please generate a new one.

Reply:
Fixed in commit bf9abef Resolved relative paths and unique id